client/src/modules/stock/entry/modals/lots.modal.js

```diff
@@ -124,6 +126,19 @@ function StockDefineLotsModalController(Instance, Notify, uiGridConstants, Data,
     var hasExpiration = (new Date(inventory.expiration_date) >= new Date());
     inventory.is_valid = (hasQuantity && hasLotLabel && hasExpiration);
 
+    // set the value if the date is not correct
+    vm.isExpiration = hasExpiration ? false : true;
+    vm.isExcessiveQuatity = hasQuantity ? false : true;
+
+    vm.submitError = vm.isExpiration;
+    if (vm.isExpiration) {
+      vm.errorText = 'FORM.ERRORS.PLEASE_CHECK_EXPIRY_DATE';
```

Comment:
Since these errors are only related to Stock, it probably makes sense to put them in the stock en/fr JSON.  Something like `STOCK.ERRORS.PLEASE_CHECK_EXPIRY_DATE`.